docs/updates/20220301_erase_on_upload.md

```diff
@@ -0,0 +1,22 @@
+# Set `eraseOnUpload` to `true` for React adapters
+
+Since `@iobroker/adapter-dev` version 1.1.0 the output from `npm run build` might be split into multiple output files with random names.
+
+As `ioBroker.js-controller` with versions below 4.1 will not erase old files on upload, new adapter versions might upload more and more files to the `admin` directory.
```

Comment:
```suggestion
Since all available `ioBroker.js-controller` versions do not erase old files on upload, new adapter versions might upload more and more files to the `admin` directory.
```